lib/src/rendering/gap.dart

```diff
@@ -3,54 +3,54 @@ import 'package:flutter/rendering.dart';
 
 class RenderGap extends RenderBox {
   RenderGap({
-    double mainAxisExtent,
-    double crossAxisExtent,
-    Axis fallbackDirection,
-    Color color,
+    double? mainAxisExtent,
```

Comment:
`mainAxisExtent` shouldn't be null I think, because all the widgets have a not nul `mainAxisExtent`.